views/navbar.twig

```diff
@@ -0,0 +1,36 @@
+{% set links = [
+  {'url': '/', 'title': 'Home'},
+  {'url': '/', 'title': 'About'},
+  {'url': '/', 'title': 'Help'}
```

Comment:
We lost the login/signup links in the navbar. Was this on purpose? Just curious

Reply:
Is it possible to make these urls overloadable if we pass an param from the router? Could be useful if we need to add certain dynamic urls.

Reply:
Yeah it was sorta on purpose as it kept everything simpler for now! Also, I like the idea of variables for the urls. That's the only thing that is lacking in this implementation

Reply:
👍